### ISSUE TITLE
The pop-out window doesn't work if app-db contains composite keys

### ISSUE THREAD

**Issue by p-himik** [flexsurfer/re-frisk#41]
I just tried to use https://github.com/vimsical/subgraph and it appears to be working, but it creates keys like `[:score/id 17]`. The in-app re-frisk window works fine:

![image](https://user-images.githubusercontent.com/4410314/32426111-8919c308-c2ea-11e7-91c4-a165fc1c94cb.png)
But trying to invoke the pop-out window leads to
```
Error rendering component (in re_frisk_shell.re_com.views.main > re_com.box.v_box > re_frisk_shell.re_com.splits.h_split > re_frisk_shell.re_com.views.main_view > re_com.box.v_box > re_frisk_shell.re_com.splits.v_split > re_com.box.v_box > re_frisk_shell.re_com.splits.v_split > re_com.box.v_box > re_frisk_shell.re_com.ui.scroller > re_frisk_shell.frisk.main_frisk)
```
in the JS console.

**Comment by borkdude** [comment]
+1

**Comment by borkdude** [comment]
I'm experiencing this problem too. Giving it a +1.

**Comment by flexsurfer** [comment]
@p-himik @borkdude thanks for your report, it's because of sorting, as workaround you can uncheck sorted, i'm working on this

**Comment by flexsurfer** [comment]
fixed in 0.5.3
